index.d.ts

```diff
@@ -24,6 +26,11 @@ declare namespace meow {
 		- `type`: Type of value. (Possible values: `string` `boolean` `number`)
 		- `alias`: Usually used to define a short flag alias.
 		- `default`: Default value when the flag is not specified.
+		- `isRequired`: Boolean or Function that specifies if this flag is required.
+			Two arguments are passed to the function.
+			The first arguments is the flags object, it contains the flags converted to camelCase excluding aliases.
+			The second arugment is the input string array, it contains the non-flag arguments.
+			The function should return a Boolean, true if the flag is requried, otherwise false.
```

Comment:
The function docs should be at https://github.com/sindresorhus/meow/pull/141/files#diff-b52768974e6bc0faccb7d4b75b162c99R5 and you don't have to mention the types. It's already shown by the actual types. Mention what it does and what problem it solves.

Reply:
Yeah, I was just trying to make sure that the `index.d.ts` and the `readme.md` file reflect each other as much as possible, that's why I left the type information there, because although typescript has a better way of documenting type information, in the readme file this is the only way.
Anyways I left the documentation with the type information in the readme file and updated the typescript file with a better documentation